### ISSUE TITLE
Fix remove_foreign_key call for non-standard column

### ISSUE THREAD

**Issue by joshea0** [SquaredLabs/nucore-uconn#69]
TableXI has a patch for this which is not in our fork: https://github.com/tablexi/nucore-open/pull/1691

> Running rake db:migrate on an empty database currently fails on this call to remove_foreign_key, because Rails can’t find the appropriate column. When the foreign key is added, it is added against the products table but it is on the instrument_id column, so the remove_foreign_key needs additional information to be able to find the appropriate foreign key.

**Comment by jhanggi** [comment]
Just curious, is it that your fork is out of date and needs to be updated to the latest `master`? The https://github.com/tablexi/nucore-open/tree/uconn-design-updates branch is largely up to date (looks like as of Nov 21). Or did something wonky happen in a merge somewhere?

**Comment by joshea0** [comment]
@jhanggi I've been working off of https://github.com/SquaredLabs/nucore-uconn/tree/master and experienced the issue there. I have only recently joined this project, so perhaps I made an incorrect assumption in using that branch instead of `uconn-design-updates` in `nucore-open`.

**Comment by jhanggi** [comment]
No, you're using the right thing: you should be working off of nucore-uconn/master. It's probably just about time for a merge from nucore-open/master.

**Comment by joshea0** [comment]
Ah okay thanks for the clarification. I believe you're correct, we will need to do a merge soon.
